packages/snippet-bot/package.json

```diff
@@ -30,7 +30,11 @@
   },
   "dependencies": {
     "@octokit/rest": "^18.0.0",
+    "@types/minimatch": "^3.0.3",
```

Comment:
You can move this into `devDependencies`

Reply:
done